suite/tests/linux/clone.c

```diff
@@ -154,6 +186,14 @@ main()
      */
     test_thread(true /*share_sighand*/, true /*clone_vm*/, false /*use_clone3*/);
     test_thread(true /*share_sighand*/, true /*clone_vm*/, true /*use_clone3*/);
+
+#if defined(X86)
+    /* Test passing NULL for the stack pointer (xref i#6514). */
+    test_with_null_stack_pointer(false /*clone_vm*/, false /*use_clone3*/);
```

Comment:
style: with `=` and before the value: so `/*clone_vm=*/false`.  Ditto for 2 instances on each of these 4 lines.

Reply:
Done